build/dependencies.props

```diff
@@ -7,7 +7,7 @@
     <GoogleProtobufVersion>3.1.0</GoogleProtobufVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <JsonNetVersion>10.0.1</JsonNetVersion>
-    <MoqVersion>4.7.1</MoqVersion>
+    <MoqVersion>4.7.49</MoqVersion>
```

Comment:
The specific reason we're updating Moq everywhere is that 4.7.1 causes a NuGet warning with Castle.Core, which is resolved in 4.7.49.